config/src/wasm.rs

```diff
@@ -3,7 +3,7 @@
 use iroha_config_base::derive::{Documented, Proxy};
 use serde::{Deserialize, Serialize};
 
-const DEFAULT_FUEL_LIMIT: u64 = 20_000_000;
+const DEFAULT_FUEL_LIMIT: u64 = 23_000_000;
```

Comment:
@Arjentix, threre is mismatch between value in the `config.json` and `DEFAULT_FUEL_LIMIT`.